lib/rules/class-methods-use-this.js

```diff
@@ -118,6 +126,8 @@ module.exports = {
             "FunctionDeclaration:exit": exitFunction,
             FunctionExpression: enterFunction,
             "FunctionExpression:exit": exitFunction,
+            "PropertyDefinition > ArrowFunctionExpression.value": enterFunction,
+            "PropertyDefinition > ArrowFunctionExpression.value:exit": exitFunction,
```

Comment:
Should we put this behind an option, since this isn't a class method by definition?